package/CHANGELOG

```diff
@@ -127,6 +129,7 @@ Changes
     enforced to be of the form ``[lx, ly, lz, alpha, beta, gamma]`` as required
     by the docs (Issue #2046, PR #2048)
   * Added periodic keyword to select_atoms (#759)
+  * PCA.transform now requires that PCA.run has already been called (PR #2055)
```

Comment:
Add: PCA.transform now requires that `run` has been called before, otherwise a`ValueError` is raised.